### ISSUE TITLE
backend company-capf error

### ISSUE THREAD

**Issue by ianhattendorf** [wentasah/meson-mode#16]
This occurs after opening a `meson.build` file and typing. An easy way to reproduce is to open a new blank `meson.build` and type a few characters. I have `company-rtags` and `company-ansible` working fine, any ideas?

```
Company: An error occurred in auto-begin
Company: backend company-capf error "Symbol’s value as variable is void: meson-builtin-vars" with args (prefix)
```

When pressing `C-M-i`:
```
completion--capf-wrapper: Symbol’s value as variable is void: meson-builtin-vars
```

init.el meson-mode section:
```
(use-package meson-mode
  :ensure t
  :mode "\\meson\\.build\\'"
  )
```

I've also tried:
```
(require 'meson-mode)
```

Using package v0.1, emacs v25.3.1.

**Comment by tko** [comment]
Best I can tell this only happens if meson-mode is byte-compiled; with `meson-mode.elc` I get the same error (different circumstances), works better without.

There's bunch of odd looking `eval-and-compile`s around but I don't know enough elisp to really know how they behave. What does strike me as suspicious though is that `meson-builtin-vars` is the only one wrapped in `eval-when-compile` instead.

https://github.com/wentasah/meson-mode/blob/212d9f38a08074f1cb6e914e12b60bc52dcb8bee/meson-mode.el#L102-L104

On quick try locally changing that to `eval-and-compile` like all the rest might help?

**Comment by wentasah** [comment]
Sorry for late response. I think this has been fixed in #19. I've just released new version, which includes the fix.
